server/routes/api.js

```diff
@@ -1,5 +1,6 @@
 var express = require('express');
 var router = express.Router();
+const categoryClient = require("../dbClients/categoriesDB");
```

Comment:
call it categoriesDB (not catergoryClient - normally you match your moduleName)

Reply:
alright